Marlin/language_cz.h

```diff
@@ -58,11 +61,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Zahrat PLA")
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Vse")
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Tryska")
```

Comment:
MSG_PREHEAT_1_END should be translated to "End" or "finished".
I guess "Tryska" means "Nozzle"